odd-platform-ui/src/theme/typography.ts

```diff
@@ -0,0 +1,72 @@
+import createBreakpoints from '@material-ui/core/styles/createBreakpoints';
+import createTypography from '@material-ui/core/styles/createTypography';
+import { palette } from 'theme/palette';
+
+export const breakpoints = createBreakpoints({});
+export const { pxToRem } = createTypography(palette, {});
+export const typography = createTypography(palette, {
+  h1: {
+    fontSize: pxToRem(20),
+    lineHeight: pxToRem(32),
+    fontWeight: 500,
+  },
+  h2: {
+    fontSize: pxToRem(18),
+    lineHeight: pxToRem(24),
+    fontWeight: 500,
+  },
+  h3: {
+    fontSize: pxToRem(16),
+    lineHeight: pxToRem(24),
+    fontWeight: 500,
+  },
+  h4: {
+    fontSize: pxToRem(14),
+    lineHeight: pxToRem(20),
+    fontWeight: 500,
+  },
+  h5: {
+    fontSize: pxToRem(12),
+    lineHeight: pxToRem(16),
+    fontWeight: 500,
+  },
+  h6: {
+    fontSize: pxToRem(11),
+    lineHeight: pxToRem(16),
+    fontWeight: 500,
+  },
+  subtitle1: {
+    fontSize: pxToRem(14),
+    lineHeight: pxToRem(20),
+    color: '#7A869A',
+    fontWeight: 400,
+  },
+  subtitle2: {
+    fontSize: pxToRem(12),
+    lineHeight: pxToRem(16),
+    color: '#7A869A',
+    fontWeight: 400,
+  },
+  body1: {
+    fontSize: pxToRem(14),
+    lineHeight: pxToRem(20),
+    fontWeight: 400,
+    color: '#091E42',
+    [breakpoints.down(1200)]: {
```

Comment:
This block is used in many places, would be nice to avoid copy-paste and to be able to update all breakpoints at once.